content/en/examples/pods/pod-with-toleration.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+  labels:
+    env: test
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    imagePullPolicy: IfNotPresent
+  nodeSelector:
```

Comment:
This `nodeSelector` makes it harder for a reader to try out the sample YAML (maybe they don't have any matching nodes).

*Not* a big deal, but I wonder what the line is adding.